blog/thirdweb.md

```diff
@@ -0,0 +1,260 @@
+---
+slug: thirdweb-wallet-upload
+hide_table_of_contents: false
+title: "Introducing Wallet SDK and Sending remote storage"
+date: 2023-05-17
+authors: fabri
+image: "https://blog.thirdweb.com/content/images/size/w2000/2023/05/How-to-use-smart-wallet-.png"
+description: "Sending remote attachments with XMTP"
```

Comment:
I think we can update this to the introduction text: Creating an effective 'Connect Wallet' flow is a challenging aspect of Web3 development. Thankfully, the team at Thirdweb has developed an excellent SDK, simplifying this process and enabling a superior wallet experience for your users.